tests/unit/algorithms/segmentation/test_helpers.py

```diff
@@ -0,0 +1,114 @@
+import os
+import warnings
+
+import numpy as np
+
+from otx.api.configuration.configurable_parameters import ConfigurableParameters
+from otx.api.entities.annotation import (
+    Annotation,
+    AnnotationSceneEntity,
+    AnnotationSceneKind,
+)
+from otx.api.entities.color import Color
+from otx.api.entities.dataset_item import DatasetItemEntity
+from otx.api.entities.datasets import DatasetEntity
+from otx.api.entities.image import Image
+from otx.api.entities.label import Domain, LabelEntity
+from otx.api.entities.label_schema import LabelGroup, LabelGroupType, LabelSchemaEntity
+from otx.api.entities.model import ModelConfiguration, ModelEntity
+from otx.api.entities.shapes.ellipse import Ellipse
+from otx.api.entities.shapes.polygon import Point, Polygon
+from otx.api.entities.shapes.rectangle import Rectangle
+from otx.api.entities.task_environment import TaskEnvironment
+from tests.test_helpers import generate_random_annotated_image
+
+DEFAULT_SEG_TEMPLATE_DIR = os.path.join("otx/algorithms/segmentation/configs", "ocr_lite_hrnet_18_mod2")
+
+labels_names = ("rectangle", "ellipse", "triangle")
+
+
+def create_model():
```

Comment:
I think this function can be modified as fixture because it doesn't get argument. I think that difference between use case for fixture and utility function for test code is that first one provides fixed result and second one can provide various results depending on argument. But some of functions in this file including not only this but also some of functions below provides fixed result and this can be used as fixture, which result in making them easier to use because not need to import and call them but just set them as argument. How do you think about?

Reply:
I revised based on your review. Could you check again?